src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentTransactionInfoPlugin.java

```diff
@@ -315,6 +348,38 @@ private static PaymentPluginStatus pspResultToPaymentPluginStatus(final PaymentS
         }
     }
 
+    private static List<PluginProperty> extractPluginProperties(PurchaseResult purchaseResult) {
+        ImmutableMap.Builder<String, String> builder = ImmutableMap.builder();
```

Comment:
I would not use a `ImmutableMap` here as I'm not sure `purchaseResult.getFormParameter()` has only non-null values.

Reply:
Switched to HashMap.